MyBooking.Domain/Interfaces/IAdvertRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using MyBooking.Domain.Entities;
+using System.Collections.Generic;
+
+namespace MyBooking.Domain.Interfaces
+{
+    public interface IAdvertRepository
+    {
+        List<Advert> GetAll();
+        Advert GetById(int id);
+        Advert GetByAddress(string address);
+        List<Advert> GetWithFilters(bool animals = false, int countPeople = 0, int minCost = 0, int maxCost = 0);
```

Comment:
How about making default value for count of people 1? 0 is strange...

Reply:
This type of value means that this argument can be skipped, if 0, then this filter is not used

Reply:
Okay